ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Solaris.yml

```diff
@@ -0,0 +1,161 @@
+---
+###########
+# Solaris #
+###########
+
+- name: Set sudoers file
+  set_fact:
+    Sudoers_File: "/etc/opt/csw/sudoers"
+  tags: [jenkins_user, adoptopenjdk]
+
+- name: Check if pkgutil is installed
+  shell: pkgutil --version
+  ignore_errors: true
+  register: pkgutil_installed
+  tags:
+    - build_tools
+
+- name: Install pkgutil
+  shell: |
+    set -o pipefail
+    echo action=nocheck > responses.txt
+    yes all | pkgadd -d http://get.opencsw.org/now -a responses.txt
+    /opt/csw/bin/pkgutil -U
+    /opt/csw/bin/pkgutil -y -i pkgutil
+    /usr/sbin/pkgchk -L CSWpkgutil
```

Comment:
What were these two lines for? Debugging? If we're downloading the latest in the initial `pkgadd` step this seems unnecessary

Reply:
removed